web/themes/new_weather_theme/assets/sass/components/weather-story.scss

```diff
@@ -1,16 +1,74 @@
 @use "uswds-core" as *;
 
-@include at-media-max("tablet") {
-  // Below the tablet breakpoint, weather story images should be the full width
-  // of the screen. The image's container has 2 units of padding on each side,
-  // so the image's max width needs to be 100% + 4 units to overcome that.
-  [data-wx-weather-story] img {
-    max-width: calc(100% + units(4));
-  }
-}
-
-/* Weather story overrides
-------------------------------------- */
-#wx-weather-story-content h3 {
-    font-weight: normal;
+/**
+ * The Weather Story area uses a simple CSS Grid
+ * API setup in order to facilitate the reordering and
+ * display of the story image when at the desktop size
+ */
+.wx-weather-story-wrapper {
+    display: grid;
+    grid-template-areas:
+        "a"
+        "b"
+        "c";
+}
+
+.wx-weather-story-wrapper > *:first-child {
+    grid-area: a;
+}
+
+.wx-weather-story-wrapper > *:nth-child(2) {
+    grid-area: b;
+}
+
+.wx-weather-story-wrapper > :last-child {
+    grid-area: c;
+}
+
+.wx-weather-story-block-section {
+    position: relative;
+}
+
+/**
+ * NOTE: We use the before/after pseudo elements
+ * to make the colored weather story section "appear"
+ * to be full screen width on tablet/desktop (the current
+ * layout structure prevents us from doing this a easier way).
+ * These pseudo elements _do not_ respect the border-box
+ * sizing of their parent element, so we have to add pixels
+ * to the height and the top attributes
+*/
+.wx-weather-story-block-section::before,
+.wx-weather-story-block-section::after {
+    content: " ";
+    display: block;
+    position: absolute;
+    width: units(2);
+    height: calc(100% + 2px);
+    background-color: color("accent-cool-lightest");;
+    border-top: 1px solid color("base-light");
+    border-bottom: 1px solid color("base-light");
+    top: -1px;
+}
+
+.wx-weather-story-block-section::before {
+    right: 100%;
+}
+
+.wx-weather-story-block-section::after {
+    left: 100%;
+}
```

Comment:
Yesterday it didn't look like this was visible on mobile, but looking at it today it does look it contributes some to the mobile horizontal scrolling issue. It may not once we fix the grid-gap issue, but flagging so we can check.